lib/services/prism-utils.js

```diff
@@ -130,9 +117,31 @@ function PrismUtils() {
     }, '');
   }
 
+  function chainMiddlewares(middlewares) {
```

Comment:
I'll be honest.  I don't really understand what's going on here or its usage earlier in this file.  Is this implementation a consequence of now using `body-parser (https://www.npmjs.com/package/body-parser)?  It looks to me like you're deleting the `req.body` if it is an empty object `{}`, is that so `body-parser` can parse the request body?

Reply:
the `json` parsing will place a blank object in `req.body` if it parses a blank request - we remove it if it is blank. the chain is so that we can use a few different parsers, one right after the other, to parse the request based on the content types.

Reply:
Ah, I got it now.  I wasn't connecting the fact you were running other parsers as well.  If the body is not URL encoded or JSON content will the other parsers gracefully skip parsing?

Reply:
Tests would have spotted that if it weren't the case.  OK.